php/class-post-type.php

```diff
@@ -447,41 +451,82 @@ public function render_data_metabox( $post ) {
 				}
 				echo '</span>';
 			}
-
-			echo '</summary>';
-
-			if ( '' === $value ) {
-				$preview = '<p><em>' . esc_html__( '(Empty string)', 'customize-snapshots' ) . '</em></p>';
-			} elseif ( is_string( $value ) || is_numeric( $value ) ) {
-				$preview = '<p>' . esc_html( $value ) . '</p>';
-			} elseif ( is_bool( $value ) ) {
-				$preview = '<p>' . wp_json_encode( $value ) . '</p>';
-			} else {
-				$preview = sprintf( '<pre class="pre">%s</pre>', esc_html( Customize_Snapshot_Manager::encode_json( $value ) ) );
+			if ( isset( $conflicts_settings[ $setting_id ] ) ) {
+				$setting_id_key = str_replace( '[', '\\[', $setting_id );
+				$setting_id_key = str_replace( ']', '\\]', $setting_id_key );
+				$title_text = sprintf( '%s ' . __( 'has potential conflicts (click to expand)', 'customize-snapshots' ), $setting_id );
```

Comment:
This is problematic to translate

Reply:
@westonruter Did you mean, use `_x()` because the same string has been used before? https://github.com/xwp/wp-customize-snapshots/pull/71#discussion_r155712820
or using `(` `)` is problematic?

Reply:
What if a language is Object-Verb-Subject order? The placeholder should be inside the translation string.

Reply:
Great point @westonruter  💯